AmazonChimeSDKDemo/AmazonChimeSDKDemo/Base.lproj/Main.storyboard

```diff
@@ -961,6 +962,65 @@
+                                <rect key="frame" x="20" y="286" width="354" height="34"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="14"/>
+                                <textInputTraits key="textInputTraits"/>
+                            </textField>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="7dQ-v6-lgr"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        <constraints>
+                            <constraint firstItem="W4e-99-NSd" firstAttribute="top" secondItem="HAa-I4-bUz" secondAttribute="bottom" constant="10" id="8Qj-3A-ZJq"/>
+                            <constraint firstItem="HAa-I4-bUz" firstAttribute="top" secondItem="UPV-0s-G4L" secondAttribute="bottom" constant="10" id="Iz9-KS-6jk"/>
+                            <constraint firstItem="W4e-99-NSd" firstAttribute="top" secondItem="HAa-I4-bUz" secondAttribute="bottom" constant="124" id="Mhl-c8-RZj"/>
+                            <constraint firstItem="7dQ-v6-lgr" firstAttribute="trailing" secondItem="W4e-99-NSd" secondAttribute="trailing" constant="30" id="Z9i-mE-XBR"/>
+                            <constraint firstItem="W4e-99-NSd" firstAttribute="leading" secondItem="7dQ-v6-lgr" secondAttribute="leading" constant="30" id="joS-3o-6N2"/>
+                            <constraint firstItem="W4e-99-NSd" firstAttribute="top" secondItem="HAa-I4-bUz" secondAttribute="bottom" constant="30" id="o11-4Q-n4C"/>
+                            <constraint firstItem="7dQ-v6-lgr" firstAttribute="bottom" secondItem="W4e-99-NSd" secondAttribute="bottom" constant="350" id="sOv-dF-dyv"/>
+                            <constraint firstItem="7dQ-v6-lgr" firstAttribute="trailing" secondItem="UPV-0s-G4L" secondAttribute="trailing" constant="30" id="tdx-2f-MmU"/>
+                            <constraint firstItem="UPV-0s-G4L" firstAttribute="leading" secondItem="7dQ-v6-lgr" secondAttribute="leading" constant="30" id="tyG-mf-hd8"/>
+                            <constraint firstItem="7dQ-v6-lgr" firstAttribute="trailing" secondItem="HAa-I4-bUz" secondAttribute="trailing" constant="30" id="zGQ-Ug-1V0"/>
+                            <constraint firstItem="HAa-I4-bUz" firstAttribute="leading" secondItem="7dQ-v6-lgr" secondAttribute="leading" constant="30" id="zc6-7g-W3Z"/>
+                        </constraints>
+                    </view>
+                    <connections>
+                        <outlet property="engineTextField" destination="UPV-0s-G4L" id="BwK-EG-N6q"/>
+                        <outlet property="languageTextField" destination="HAa-I4-bUz" id="j3N-1M-j6v"/>
```

Comment:
Wouldn't it be great if there is some default value so that people can only change value they want? Refer to the JS demo.